README.md

```diff
@@ -90,7 +90,7 @@ jake help             # List the above commands.
 ## Usage
 
 ```bash
-node built/local/tsc.js hello.ts
+tsc hello.ts

```

Comment:
AFAICT this is meant for contributors how to execute a locally built version of the compiler

Reply:
I see. It is just that the readme's "Installation" recommends `npm install -g typescript`, but the "Usage" section refers to contributors.

Also this readme is published then on https://www.npmjs.com/package/typescript where it doesn't really make sense.

close drop this PR then?